renderlib/AxisHelperTool.cpp

```diff
@@ -1,5 +1,7 @@
 #include "AxisHelperTool.h"
 
+#include "AppScene.h"
+
```

Comment:
I honestly can't remember right now but it could be because of changes in AppScene or things that AppScene includes, too.
Sometimes it's easy to just include extra header files that never get used.  The compiler doesn't really tell you about that - you have to use static analysis tools to clean up extra unnecessary includes.